static_src/components/users.jsx

```diff
@@ -24,7 +24,14 @@ const ORG_MANAGER = 'org_manager';
 const SPACE_MANAGER = 'space_manager';
 const ORG_ENTITY = 'organization';
 const SPACE_ENTITY = 'space';
-const cfCliLink = 'https://docs.cloudfoundry.org/adminguide/cli-user-management.html#space-roles';
+const ORG_INVITE_HELP = 'Only an Org Manager can new invite users to this ' +
```

Comment:
In some cases template strings introduce unnecessary spaces, and will fail tests if the strings aren't formatted identically. 

If the strings are all moved into constant files, the issue should go away?